run_service.sh

```diff
@@ -28,7 +28,7 @@ wei_to_dai() {
     local wei="$1"
     local decimal_precision=4  # Change this to your desired precision
     local dai=$($PYTHON_CMD -c "print('%.${decimal_precision}f' % ($wei / 1000000000000000000.0))")
-    echo "$dai DAI"
+    echo "$dai"
```

Comment:
Why?

Reply:
"Functions should do only one thing":

The method was doing 2 things: conversion and formatting.

Now formatting is done in the strings using this function (I added DAI to these strings).

This allows using the output of this function as an integer directly, in case it is ever required.